src/python/CRABClient/Commands/uploadlog.py

```diff
@@ -27,28 +24,25 @@ def __call__(self):
         self.logger.debug("uploadlog started")
         taskname = None
         #veryfing the log file exist
-        if self.options.logpath is not None:
-            logfilename = str(time.strftime("%Y-%m-%d_%H%M%S"))+'_crab.log'
-            self.logfile = self.options.logpath
-        elif os.path.isfile(self.logfile):
+        if os.path.isfile(self.logfile):
             self.logger.debug("crab.log exists")
             try:
                 taskname = self.cachedinfo['RequestName']
                 logfilename = str(taskname)+".log"
-            except:
-                self.logger.info("Couldn't get information from .requestcache (file likely not created due to submission failure), try\n"
-                        "'crab uploadlog --logpath=<path-to-log-file-in-project-dir>'")
-                return
+            except Exception:
+                self.logger.info("Couldn't get information from .requestcache (file likely not created due to submission failure),\n" +
+                                 "Please local crab.log yourself and copy/paste into the mail to support if needed")
```

Comment:
I don't fully understand wording used this message. maybe we should change it to `Please copy/paste crab.log into the mail yourself and send to support if needed.`

Reply:
oops typo

Reply:
please check if message is clear in new version. Thanks

Reply:
looks good, thanks